pkg/storage/filesystem/btrfs.go

```diff
@@ -97,6 +97,51 @@ func (b *btrfs) Create(ctx context.Context, name string, policy pkg.RaidProfile,
 	return newBtrfsPool(name, devices, &b.utils), nil
 }
 
+func (b *btrfs) CreateForce(ctx context.Context, name string, policy pkg.RaidProfile, devices ...*Device) (Pool, error) {
```

Comment:
Can we also refactor the code a bit to avoid having to copy the full function like this ?

Reply:
Something like:
```go
 func (b *btrfs) CreateForce(ctx context.Context, name string, policy pkg.RaidProfile, devices ...*Device) (Pool, error) { 
    return b.create(...,true)
}

 func (b *btrfs) Create(ctx context.Context, name string, policy pkg.RaidProfile, devices ...*Device) (Pool, error) { 
   return b.create(...,false)
}

 func (b *btrfs) create(ctx context.Context, name string, policy pkg.RaidProfile, devices ...*Device, force bool) (Pool, error) { 
...
}
```